src/PrestaShopBundle/Form/Admin/Product/ProductShipping.php

```diff
@@ -150,6 +150,7 @@ public function buildForm(FormBuilderInterface $builder, array $options)
                 'weight',
                 FormType\NumberType::class,
                 [
+                    'scale' => self::PRESTASHOP_WEIGHT_DECIMALS,
```

Comment:
```suggestion
                    'scale' => static::PRESTASHOP_WEIGHT_DECIMALS,
```

Reply:
just a little comment:
```php
<?php

class A
{
    public const B = 'test';

    public function __construct()
    {
        var_dump(self::B);
        var_dump(static::B);
    }
}

class C extends A
{
    public const B = 'here';
}


new C();
```
It will display:
```
/home/got/test.php:9:
string(4) "test"
/home/got/test.php:10:
string(4) "here"
```